docker/php/php-fpm.d/zz-docker.conf

```diff
@@ -12,4 +12,3 @@ pm.max_children = 25
 pm.start_servers = 10
 pm.min_spare_servers = 5
 pm.max_spare_servers = 20
-pm.max_requests = 500
```

Comment:
seemed like a good idea, but does not seem to work that well...

pm.max_requests [int](https://www.php.net/manual/en/language.types.integer.php)
    The number of requests each child process should execute before respawning. This can be useful to work around memory leaks in 3rd party libraries. For endless request processing specify '0'. Equivalent to PHP_FCGI_MAX_REQUESTS. Default value: 0.